assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -873,6 +877,10 @@
      * Site property for forcing anonymous grading in a site
      */
     private static final String SAK_PROP_FORCE_ANON_GRADING = "assignment.anon.grading.forced";
+    /** 
+     * Sakai.property for enable/disable the group filter
+     */
+    private static final String SAK_PROP_ENABLE_GROUP_FILTER = "assignment.group.filter.enabled";
```

Comment:
This property should be in AssignmentServiceConstants.

Reply:
Good to know, thanks!